FirebaseAppDistribution/Sources/Public/FIRAppDistribution.h

```diff
@@ -15,51 +15,33 @@
 #import <AppAuth/AppAuth.h>
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
+#import "FIRAppDistributionRelease.h"
 
-NS_ASSUME_NONNULL_BEGIN
-
-/**
- * The release information returned by the update check when a new version is available.
- */
-NS_SWIFT_NAME(AppDistributionRelease)
-@interface FIRAppDistributionRelease : NSObject
-
-// The short bundle version of this build (example 1.0.0)
-@property(nonatomic, copy) NSString *displayVersion;
-
-// The build number of this build (example: 123)
-@property(nonatomic, copy) NSString *buildVersion;
+#define STR(x) STR_EXPAND(x)
+#define STR_EXPAND(x) #x
```

Comment:
These should not be part of the public API.

Reply:
sorry about that - done!